sklearn/linear_model/least_angle.py

```diff
@@ -1145,10 +1154,14 @@ def fit(self, X, y):
         # Now compute the full model
         # it will call a lasso internally when self if LassoLarsCV
         # as self.method == 'lasso'
-        Lars.fit(self, X, y)
+        Lars._fit(self, X, y, max_iter=self.max_iter, alpha=best_alpha,
```

Comment:
Is it the case that previously the call to Lars.fit reverted to a call to lasso_lars with `alpha_min=0`, while now `alpha_min=best_alpha`?